tests/unit/jwt-verifier.test.ts

```diff
@@ -1126,6 +1151,29 @@ describe("unit tests jwt verifier", () => {
         })
       ).resolves.toMatchObject({ hello: "world" });
     });
+    test("happy flow - ES256", () => {
+      const es256keypair = generateKeyPair({
+        kty: "EC",
+        alg: "ES256",
+      });
+      const issuer = "https://example.com";
+      const audience = "1234";
+      const signedJwt = signJwt(
+        { alg: "ES256", kid: keypair.jwk.kid },
+        { aud: audience, iss: issuer, hello: "world" },
```

Comment:
We test with a `validTokenPayload` having unicode characters in the `tests/vite-app/util/generateExampleTokens.ts` , but should we also do so in the unit tests?

```javascript
hēłłœ: "wørłd"
```